source/styles/index.styl

```diff
@@ -0,0 +1,70 @@
+[markdown-page="index"]
```

Comment:
`.page-<page>` is a much faster selector. no need to use attr selectors unless javascript is modifying it. :)